### PR TITLE
Added Stream #107

### PR DESCRIPTION
**Pull request by weinzierl-florian**
I added all the videos from stream nr. 107

### CODE REVIEW

**Comment by shiffman** [comment]
Looks great to me, thank you! Will wait for @nilsweb to merge or comment.

File: _Courses/programming-with-text/11-chrome-extensions/11.8-cross-browser-extensions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Cross-browser Extensions"
+video_number: 78
```
Comment (nilsweb) [inline _Courses/programming-with-text/11-chrome-extensions/11.8-cross-browser-extensions.md]: This should be `video_number: 8`

File: _Courses/programming-with-text/11-chrome-extensions/11.8-cross-browser-extensions.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+date: 2017-11-28
+video_id: RJtjp-Vtwjs
+repository: /A2Z/session11/11-8_Cross-Browser_Extensions
```
Comment (nilsweb) [inline _Courses/programming-with-text/11-chrome-extensions/11.8-cross-browser-extensions.md]: This links to nowhere. It should be `/A2Z/session11/11-08_Cross-Browser_Extensions`

File: _Streams/107-chrome-extensions-continued.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  - title: "Code Newbie Podcast"
+    url: "https://www.codenewbie.org/"
+  - title: "Adam Blau on Twitter"
+    url: "https://twitter.com/adamblau"
+
+videos:
+  - title: "Node.js tutorial"
+    url: "/Tutorials/15-twitter-bot/15.1-intro-nodejs"
+  - title: "Maze Generator"
+    url: "/_CodingChallenges/10.1-maze-dfs-p5"
```
Comment (nilsweb) [inline _Streams/107-chrome-extensions-continued.md]: Small typo: Don't use the underscore here.

**Comment by shiffman** [comment]
Thanks all!

**Pull request merged**